pages/internship.js

```diff
@@ -0,0 +1,116 @@
+import Link from 'next/link';
+import React from 'react';
+
+import Faq from 'react-faq-component';
+import Carousel from 'react-multi-carousel';
+
+import Banner from '../components/Banner';
+import CommitteeInternshipInfo from '../components/Internship/CommitteeInternshipInfo';
+import InternshipTimeline from '../components/Internship/InternshipTimeline';
+import NextSteps from '../components/Internship/NextSteps';
+import Layout from '../components/Layout';
+
+import data from '../data';
+import 'react-multi-carousel/lib/styles.css';
+
+function InternshipPage(){
+    const { committees, internship } = data;
+    const { items, testimonials, QA, FAQStyles, responsive } = internship;
+    return(
+        <Layout>
+            <Banner decorative />
+            <div className="content-section text-center">
+                <h1>Internships</h1>
+                What do you do as an intern? Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tristique sollicitudin
+                nibh sit amet commodo nulla. Amet facilisis magna etiam tempor.
+            </div>
+            <CommitteeInternshipInfo
+                committees={committees}
+            />
+            <div className="content-section text-center">
+                <h1>Applications</h1>
+                <InternshipTimeline items={items}/>
+                <div className="application-information">
+                    <div>
```

Comment:
What is this `<div>` used for?

Reply:
`application-information` is a grid, so this is one of the columns!

Reply:
Gotcha. Might be good to try to incorporate a `<section>` somehow in this hierarchy, there's a ton of nested divs! That being said, let's say that this isn't a blocking concern for now.